src/Microsoft.DotNet.Darc/tests/Microsoft.DotNet.Darc.Tests/inputs/NugetConfigFiles/RemovingAllSourcesKeepsTheComments/NuGet.output.config

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This test case exercises being called with no managed package sources where there were some defined previously.
+     Expected behaviors:
+       - All managed source are removed from inside the packageSources block
+       - No comments are removed (trying to avoid merge issues), including for the not-updated 'please-leave-this-alone' repo
+       - 'darc-' entries that were not part of the update in the disabledPackageSources section are left alone.
+       - Disabled Sources are left alone (no sources included)
+ -->
+<configuration>
+  <packageSources>
+    <clear />
+    <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
+    <!--  Begin: Package sources from dotnet-arcade -->
+    <!--  End: Package sources from dotnet-arcade -->
+    <!--  Begin: Package sources from please-leave-this-alone -->
+    <!--  End: Package sources from please-leave-this-alone -->
+    <!--  Begin: Package sources from dotnet-runtime -->
+    <!--  End: Package sources from dotnet-runtime -->
+    <!--End: Package sources managed by Dependency Flow automation. Do not edit the sources above.-->
+    <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
+    <add key="dotnet-coreclr" value="https://dotnetfeed.blob.core.windows.net/dotnet-coreclr/index.json" />
+    <add key="dotnet-windowsdesktop" value="https://dotnetfeed.blob.core.windows.net/dotnet-windowsdesktop/index.json" />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+  </packageSources>
+  <disabledPackageSources>
+    <add key="dotnet-coreclr" value="true" />
+    <clear />
+    <add key="nuget.org" value="true" />
+    <clear />
+    <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
+    <!--  Begin: Package sources from dotnet-arcade -->
+    <add key="darc-int-dotnet-arcade-b043797" value="true" />
```

Comment:
@mmitche given these feeds were removed, do they need to be removed from disabled?  Disabling a non-existent key is a no-op.  If I should walk them and delete them I can, it just doesn't change much.

Reply:
Yes, they should be removed. If they aren't, then over time the number of disabled sources tends to get very large.